src/lazy/expanded/encoding_module.rs

```diff
@@ -16,13 +16,7 @@ impl EncodingModule {
             symbol_table,
         }
     }
-    pub fn ion_1_1_system_module() -> Self {
-        Self::new(
-            String::from("$ion"),
-            MacroTable::with_system_macros(),
-            SymbolTable::new(IonVersion::v1_1),
-        )
-    }
+
```

Comment:
🪧 This was unused and, in light of the singleton instance, no longer makes sense.